project-xml-data-interchange/task-elements-and-xml-structure.md

```diff
@@ -326,11 +326,11 @@ Many of the Task elements represent data fields in Microsoft Office Project. For
 
 ## See Also
 
-#### Reference
+### Reference
 
-[TimephasedDataType Elements and XML Structure](https://docs.microsoft.com/office-project/xml-data-interchange/timephaseddatatype-elements-and-xml-structure?view=project-client-2016)
+[TimephasedDataType Elements and XML Structure](/office-project/xml-data-interchange/timephaseddatatype-elements-and-xml-structure?view=project-client-2016)
```

Comment:
Not sure if the versioning on the link (`?view=project-client-2016`) is necessary here. Since it's flagging a validation error, I'd consider removing it.

Reply:
Fixed. Thanks!